driver/pace/driver/diagnostics.py

```diff
@@ -131,3 +156,26 @@ def store_grid(
         self, grid_data: pace.util.grid.GridData, metadata: QuantityMetadata
     ):
         pass
+
+
+def _compute_column_integral(
+    name: str, q_in: pace.util.Quantity, delp: pace.util.Quantity
```

Comment:
should this specifically be q and delp or should these be more generic?

Reply:
This only works for column integrating tracers, for other column integrated variable, other factors are needed. For now, we only support column integrated tracers.